.pipelines/stages/jobs/steps/utils/download-huggingface-model.yml

```diff
@@ -28,7 +28,7 @@ steps:
     - powershell: |
         python -m pip install "huggingface_hub[cli]"
         huggingface-cli login --token $env:HF_TOKEN
-        huggingface-cli download ${{ parameters.HuggingFaceRepo }} --include ${{ parameters.RepoFolder }}/* --local-dir ${{ parameters.LocalFolder }} --local-dir-use-symlinks False
+        huggingface-cli download ${{ parameters.HuggingFaceRepo }} --include ${{ parameters.RepoFolder }}/* --local-dir "\\?\${{ parameters.WorkingDirectory }}\\${{ parameters.LocalFolder }}" --local-dir-use-symlinks False
```

Comment:
Why do you need to use an UNC path? Could you please write a comment to explain?